src/web_backend.py

```diff
@@ -50,7 +50,15 @@ def __init__(self, config):
 			self.subdirs = config["use_subdirs"]
 			# Used by download_and_serve():
 			self.webpath = tempfile.mkdtemp()
-			self.baseurl = "http://%s:%d" % (extern, port)
+			#SSL
+			self.ssl == config["ssl"]
```

Comment:
ok this is epic

Reply:
HEY IM NOT A PROFESSIONAL
atleast recommend me somthing

Reply:
`==` is a comparison not an assigment
your `if` statements are also missing colons